### DISCUSSION TITLE
How do you actually clear the terminal while using chisel

### DISCUSSION THREAD

**Discussion by salieflewis** [foundry-rs/foundry#4079] [Help]
I don't necessarily want to create a new session, I just want to clear the window like I normally would be able to with the `clear` command. 

My first instinct was to use `!e clear` but that doesn't seem to work.

**Comment by mds1** [comment]
If you run `!h` or `!help` you can see all available commands. In particular it sounds like `!c` or `!clear` is what you're looking for

**Reply by mds1** [reply]
Oh sorry I misread the question, I think that actually clears the source which you don't want 😅 cc @clabby, is this feature implemented?

**Reply by clabby** [reply]
Ctrl+L works in Chisel, but it's not clearly documented. Maybe we should add it to the book?

**Reply by mds1** [reply]
Might me nice to have a dedicated `!cs` or !clearscreen command so it shows up in the help menu with the same syntax as the other commands

**Reply by salieflewis** [reply]
Ctrl + L is exactly what I was looking for, thanks all!

**Comment by DaniPopes** [comment]
The `Ctrl + L` code is [supported in rustyline](https://github.com/kkawakam/rustyline/blob/ae70c0b725c5e9cfba7a9002ea3d6ba700bd381a/src/keymap.rs#L576), but I guess this can also be implemented within Chisel itself as a standalone command
